# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hotplug++/make.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./build/bottom.make.vanilla	2006-09-12 12:08:58.000000000 +0200
+++ ./build/bottom.make	2006-09-12 12:09:13.000000000 +0200
@@ -57,10 +57,10 @@
 
 $($(X_MODULE)_OUTPUT)/$(BINARY)$(X_LIBEXT): $($(X_MODULE)_OBJS)
 	@echo '  LINK LIB  $@'
-	$(Q)$(LD) -r -o '$@' $^ 2> /dev/null
+#	$(Q)$(LD) -r -o '$@' $^ 2> /dev/null
 #	# no AR anymore due to static initilizers
-#	$(Q)$(AR) r '$@' $^ 2> /dev/null
-#	$(Q)$(RANLIB) '$@'
+	$(Q)$(AR) r '$@' $^ 2> /dev/null
+	$(Q)$(RANLIB) '$@'
 
 $($(X_MODULE)_OUTPUT)/$(BINARY)$(X_DYNEXT): $($(X_MODULE)_OBJS)
 	@echo '  LINK DYN  $@'
